tests/func/test_suspended_frames.py

```diff
@@ -0,0 +1,29 @@
+import sys
```

Comment:
The last thing is that a unit-test like this should be on the `pydevd` side of things (so, you can just add a new test to `/ptvsd/src/ptvsd/_vendored/pydevd/tests_python/test_suspended_frames_manager.py` instead of creating a new file -- I'll take care of backporting to `pydevd` so that it runs properly as `pydevd` tests don't run on the `ptvsd` repo).

Otherwise, it should be a functional test which actually uses `ptvsd`.